packages/config/src/detect/bundleConfig.ts

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs-extra'
+import path from 'path'
+
+function detectConfig (projectRoot: string, dirs: string[], files: string[]): string | undefined {
+  for (const dir of dirs) {
+    for (const f of files) {
+      const p = path.join(projectRoot, dir, f)
+
+      if (fs.existsSync(p)) {
```

Comment:
This is running on the server - Cypress should not log for this? We use `fs.existsSync` in a lot of places.

Reply:
We need to fix this, we should be allowed to use `existsSync`, just not `readFileSync`. The fix for this for now is to `import from 'fs'` rather than `import from 'fs-extra'` when you're using `existsSync`

Reply:
I'll use the work-around for now. I had a feeling Cypress was incorrect here, not me.